### PR TITLE
Fix astraoneBidAdapter

### PR DESCRIPTION
**Pull request by lizakobrazova**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Fixed astraoneBidAdapter not working due to changes in api; added examples of usage.

- test parameters for validating bids
```
{
  bidder: "astraone",
  params: {
    placement: "inImage",
    placeId: "5f477bf94d506ebe2c4240f3",
    imageUrl: "https://creative.astraone.io/files/default_image-1-600x400.jpg"
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: integrationExamples/gpt/astraone_in-image_example.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```
Comment (Rothalack) [inline integrationExamples/gpt/astraone_in-image_example.html:1]: While I appreciate these integration examples to quickly test, I don't think you should include these intergrationExamples in the PR. It's not a norm that has been followed and is not necessary.

File: modules/astraoneBidAdapter.md [hunk 2]
```diff
@@ -109,90 +112,91 @@ var adUnits = [{
...
+        .addService(googletag.pubads());
+
+      googletag.pubads().enableSingleRequest();
+      googletag.enableServices();
+    });
+  </script>
 </head>
 <body>
-	<h2>Prebid.js Banner Ad Unit Test</h2>
+<h2>Prebid.js InImage Banner gbt Test</h2>
```
Comment (Rothalack) [inline modules/astraoneBidAdapter.md:189]: Happened to notice this reading through. Should this be gpt not gbt?

File: integrationExamples/gpt/astraone_in-image_example.html (hunk 1 above)
Comment (lizakobrazova) [reply integrationExamples/gpt/astraone_in-image_example.html]: Got it. Removed

File: modules/astraoneBidAdapter.md (hunk 2 above)
Comment (lizakobrazova) [reply modules/astraoneBidAdapter.md]: Renamed

**Comment by Rothalack** [comment]
Looks good!

**Pull request merged**